docs/core/metrics.md

```diff
@@ -376,6 +378,8 @@ If you prefer setting environment variable for specific tests, and are using Pyt
         ...
     ```
 
+#### TODO - below statement is related to Clearing metrics section ?
+
```

Comment:
@heitorlessa  a quick breakpoint to capture your attention when you will review the work

Reply:
That one is related to `Environment variables` - Metric Namespace is mandatory or else metrics don't get created. This part is to mention they can ignore env vars If they explicitly set `Metrics(namespace="X")` as part of the constructor. 

I'll play around with the words to see where it might read best

Reply:
Moved it up - also took time to fix all single `<code>` to be bold excluding code blocks as that would confuse people

![image](https://user-images.githubusercontent.com/3340292/147747007-3bc91783-7f25-4c5a-9f18-38a3ef4baf9b.png)